temba/channels/models.py

```diff
@@ -2013,10 +2013,8 @@ def send_vumi_message(cls, channel, msg, text):
         is_ussd = channel.channel_type in Channel.USSD_CHANNELS
         channel.config['transport_name'] = 'ussd_transport' if is_ussd else 'mtech_ng_smpp_transport'
 
-        if msg.response_to_id:
-            in_reply_to = Msg.objects.get(pk=msg.response_to_id).external_id
-        else:
-            in_reply_to = None
+        in_reply_to = Msg.objects.values_list(
+            'external_id', flat=True).get(pk=msg.response_to_id)
```

Comment:
@nullcode Using 'only', at least as below turns into two queries which is kinda weird:

```python
Msg.objects.filter(id=42).only('external_id').first()
```

```sql
SELECT "msgs_msg"."id", "msgs_msg"."external_id" FROM "msgs_msg" WHERE "msgs_msg"."id" = 42 ORDER BY "msgs_msg"."created_on" DESC, "msgs_msg"."id" DESC LIMIT 1
SELECT "msgs_msg"."id", "msgs_msg"."text" FROM "msgs_msg" WHERE "msgs_msg"."id" = 42
```

But you are paying the overhead of Django creating the object, which isn't insignificant in tight loops.